integration/ingress_test.go

```diff
@@ -0,0 +1,478 @@
+package integration
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/TykTechnologies/tyk-operator/api/v1alpha1"
+	tykClient "github.com/TykTechnologies/tyk-operator/pkg/client"
+	"github.com/TykTechnologies/tyk-operator/pkg/client/klient"
+	"github.com/TykTechnologies/tyk-operator/pkg/environment"
+	"github.com/TykTechnologies/tyk-operator/pkg/keys"
+	"github.com/matryer/is"
+	v1 "k8s.io/api/core/v1"
+	networkingv1 "k8s.io/api/networking/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/apimachinery/pkg/util/version"
+	"sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/e2e-framework/klient/wait"
+	"sigs.k8s.io/e2e-framework/pkg/envconf"
+	"sigs.k8s.io/e2e-framework/pkg/features"
+)
+
+// minOasGwVersion represents the minimum Tyk Gateway version required to use the new Tyk OAS API Definition.
+var minOasGwVersion = version.MustParseGeneric("5.3.0")
```

Comment:
Support of OAS was added in v4.1

Reply:
However the stable version of it released at v5.3.0. Prior versions are flagged as early access. I believe we should test against the stable version of OAS